src/pods/toolbar/components/export-button/export-button.component.tsx

```diff
@@ -125,6 +125,7 @@ export const ExportButton = () => {
       label="Export"
       onClick={handleExportClick}
       className={classes.button}
+      disabled={!canvasSchema.selectedElementId}
```

Comment:
I think this won't work, just load the table, do not select any table you will get this failing.

I guess it should be checking if there is at least one table

Check with @deletidev

Reply:
Exactly, Braulio is right. Please @MarcosApodaca check this file: https://github.com/Lemoncode/mongo-modeler/blob/vnext/src/pods/toolbar/components/save-button/save-button.component.tsx. Desired behaviour is quite similar. If you have any questions, send me a message in slack 😉